schema/release-schema.json

```diff
@@ -2283,6 +2293,26 @@
           ]
         }
       }
+    },
+    "Link": {
+      "title": "Link",
+      "description": "A typed connection to a related resource.",
```

Comment:
Where is this wording from? "A reference to a related resource" might be easier to interpret.

Reply:
It's from https://datatracker.ietf.org/doc/html/rfc5988#section-3. I'm happy with either wording.